cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest

```diff
@@ -10,7 +10,12 @@
  {% set feature_gates = "--feature-gates=" + grains.feature_gates -%}
 {% endif -%}
 
-{% set params = params + log_level + " " + feature_gates -%}
+{% set scheduling_algorithm_provider = "" -%}
+{% if grains.scheduling_algorithm_provider is defined -%}
+ {% set scheduling_algorithm_provider = "--algorithm-provider=" + grains.scheduling_algorithm_provider -%}
+{$ endif -%}
+
+{% set params = params + log_level + " " + feature_gates + " " + scheduling_algorithm_provider -%}
```

Comment:
This whole PR makes possible to configure algorithm provider in scheduler, but I'm not 100% we really need those parameters. What we want is to configure it based on the fact if autoscaler is on or not, for which we already have env-variable, right?